.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+on:
+  push:
+    branches: [main]
```

Comment:
This is awesome!

Just wondering if this won't trigger an indefinite loop?

Triggers on push to main, but the script itselfs pushes to main.